source/features/clean-footer.css

```diff
@@ -1,20 +1,20 @@
-body > .footer {
+.footer {
```

Comment:
If looking at the blame there was no reason for `body` to exist, then it's ok. Basically the concern should be whether `.footer` is used elsewhere, for unrelated widgets. This is especially a problem because this CSS is global.

Reply:
`body >` has been there since it was added in #879. I highly doubt GitHub will use this class for anything else.